tensorflow/training/docker/2.2.1/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,173 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
```

Comment:
Does 2.2.1 introduce any changes that are not backwards compatible w/ 2.2.0? If so, this version should be incremented. Else, we should remove the 2.2.0 dockerfile with major version 1

Reply:
The 2.2.1 is a security fix from TF. It is backwards compatible. The dockerfiles are copied from 2.2.0